tools/topology/topology2/avs-tplg/tplg-targets.cmake

```diff
@@ -2,10 +2,7 @@
 
 # Array of "input-file-name;output-file-name;comma separated pre-processor variables"
 list(APPEND TPLGS
-# CAVS HDMI only topology with passthrough pipelines
-"sof-hda-generic\;sof-hda-generic-idisp\;DEEPBUFFER_FW_DMA_MS=100"
-# CAVS HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
-"sof-hda-generic\;sof-hda-generic\;HDA_CONFIG=mix,DEEPBUFFER_FW_DMA_MS=100"
```

Comment:
why?

Reply:
Otherwise these would clash in the next commit with the identical ones in sof-ace-tplg/